zk-sdk/src/encryption/grouped_elgamal.rs

```diff
@@ -216,6 +220,51 @@ impl<const N: usize> GroupedElGamalCiphertext<N> {
     }
 }
 
+// Define specific grouped ElGamal ciphertext types for 2 and 3 handles since
+// `wasm_bindgen` do not yet support the export of types that take on generic
+// type parameters.
+#[cfg(target_arch = "wasm32")]
```

Comment:
nit: Since you have the `target_arch` cfg here, do you still need it on everything inside it? It would at least simplify the code a little bit to remove those `#[cfg_attr(...` and change them to `#[wasm_bindgen]`

Reply:
Good point! Updated.